pkg/controller/inferenceservice/resources/istio/virtualservice.go

```diff
@@ -184,19 +221,35 @@ func (r *VirtualServiceBuilder) CreateVirtualService(isvc *v1alpha2.InferenceSer
 	if len(explainRouteDestinations) > 0 {
 		explainRoute := istiov1alpha3.HTTPRoute{
 			Match: []istiov1alpha3.HTTPMatchRequest{
-				istiov1alpha3.HTTPMatchRequest{
+				{
+					URI: &istiov1alpha1.StringMatch{
+						Prefix: constants.ExplainPrefix(isvc.Name),
+					},
+					Authority: &istiov1alpha1.StringMatch{
```

Comment:
Why do we need Authority? Maybe add a comment to explain its use?

Reply:
we need authority to determine which gateway to use,
for internal host it goes through istio local gateway
for external host it goes through istio ingress gateway